### ISSUE TITLE
Ability to resume stopped programs

### ISSUE THREAD

**Issue by gshep** [gear-tech/gear#2564]
### Problem to Solve

#2301  introduces pausing programs mechanism, #2509 adds ability to extend rent period. The intention of this task is to add extrinsic `resume_program` to provide ability to users to run programs again.

### Possible Solution

-
after #2509

**Comment by shamilsan** [comment]
Closing as a duplicate of #2605
